dask/dataframe/indexing.py

```diff
@@ -192,6 +193,7 @@ def _loc_element(self, iindexer, cindexer):
         name = "loc-%s" % tokenize(iindexer, self.obj)
         part = self._get_partitions(iindexer)
 
+        breakpoint()
```

Comment:
```suggestion
```

Reply:
gah!

Reply:
No worries -- see https://github.com/dask/dask/pull/9638 for a neat `pre-commit` hook I ran across recently